docs/src/dedicated-gen-3/_index.md

```diff
@@ -1,79 +1,93 @@
-See the [services documentation](../../add-services/_index.md) for service-specific details.
+## Storage
 
-The [services documentation](../../add-services/_index.md) states that not every service or version available on the Grid is available on {{% names/dedicated-gen-3 %}}.
-The table below shows the currently available services and their versions for {{% names/dedicated-gen-3 %}}.
+Each {{% names/dedicated-gen-3 %}} cluster comes with 50GB of storage per environment by default.
+This storage is intended for your data (databases, search indexes, user uploaded files, etc.)
+and you can subdivide it as you want.
 
-{{< gen-3-services >}}
+You can request more storage at any time.
 
-### Local mounts
+## Differences with the Grid
 
-Because you get a redundant infrastructure, note that local mounts are local to each Virtual Machine. Since you can't know which VM is going to handle a specific request, you also don't have a guarantee regarding which local mount is going to be used. Whether you actually want to use a local mount or in fact need to set up a [Network Storage mount](../add-services/network-storage.md) depends on your specific use-case.
+Although {{% names/dedicated-gen-3 %}} adds plenty of features to your existing Grid apps,
+there are a few differences and limitations to be aware of when considering an upgrade.
 
-## Storage 
+### Available services
 
-Each Dedicated cluster comes with 50GB of storage per environment by default.
-This storage is intended for your data -- databases, search indexes, user uploaded files, and so on -- and can be subdivided in any way you want.
-50GB is only the default amount; more storage can be added as a line item in the contract and can be added at anytime that the project requires: at contract renewal or at any point in the term.
+Not every service or version available on the Grid is available on {{% names/dedicated-gen-3 %}}.
+The following table shows the currently available services and their versions for {{% names/dedicated-gen-3 %}}.
 
-Default storage is based on the default SSD block-storage offering for each cloud. Extra provisioned IOPS can be discussed with your sales representative.
+{{< gen-3-services >}}
+
+Your app can connect to each service by referencing
+the exact same [environment variables](../development/variables/_index.md) as for Grid environments.
+See the [services documentation](../../add-services/_index.md) for service-specific details.
 
-## Setting up
+### Local mounts
 
-If you are interested in Platform.sh's data cloning, environment control, and infrastructure-as-code philosophy across our supported runtimes and services, but you also need a large amount of resources and data isolation, [contact us](https://platform.sh/contact) to start setting up a {{% names/dedicated-gen-3 %}} project.
+As {{% names/dedicated-gen-3 %}} provides a redundant infrastructure,
```

Comment:
I find this paragraph a bit strange.
You have guarantees about the local mounts. They are working as they should, the only point is that they are not shared across the 3 VM. 
When a project needs a shared folder, it needs to set up a Network Storage.